uint8 led_state # смотри константы ниже (можно представить что это bool)

# led
uint8 OFF = 0
uint8 ON = 1
---
bool success   
string error_message 
